eng/Packages.Data.props

```diff
@@ -122,15 +124,15 @@
     <PackageReference Update="Apache.Avro" Version="1.11.0" />
     <PackageReference Update="CloudNative.CloudEvents" Version="2.0.0" />
     <PackageReference Update="CloudNative.CloudEvents.SystemTextJson" Version="2.0.0" />
-    <PackageReference Update="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="1.1.5" />
     <PackageReference Update="Microsoft.Azure.SignalR" Version="1.16.1" />
     <PackageReference Update="Microsoft.Azure.SignalR.Management" Version="1.16.1" />
     <PackageReference Update="Microsoft.Azure.SignalR.Protocols" Version="1.16.1" />
-    <PackageReference Update="Microsoft.Azure.SignalR.Serverless.Protocols" Version="1.9.0" />
+    <PackageReference Update="Microsoft.Azure.SignalR.Serverless.Protocols" Version="1.6.0" />
     <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.33" />
     <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.30" />
     <PackageReference Update="Microsoft.Spatial" Version="7.5.3" />
     <PackageReference Update="Newtonsoft.Json" Version="10.0.3" />
+			<PackageReference Update="Microsoft.Azure.KeyVault" Version="3.0.5" />
```

Comment:
This is not an approved dependency for T2 management libraries.

Reply:
Change reverted.